index.html

```diff
@@ -204,26 +219,28 @@ <h1>About me</h1>
             </div>
         </section>
         <section class="comments">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
-                <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment" required>
-                <hr class="InputSeparation">
-                <br>
-                <br>
-                <br>
-                <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" required>
-                <hr class="InputSeparation">
-                <textarea name="comment" id="comment" cols="30" rows="10" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here" required>Write me something...</textarea>
-                <button type="submit" class="msjButton">Get in touch</button>
-            </form>
+            <div class="coment_container">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
+                    <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment first_name" required>
+                    <hr class="InputSeparation">
+                    <br>
+                    <br>
+                    <br>
+                    <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" required>
+                    <hr class="InputSeparation">
+                    <textarea name="comment" id="comment" cols="30" rows="5" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here" required>Write me something...</textarea>
+                    <button type="submit" class="msjButton">Get in touch</button>
+                </form>
```

Comment:
- [ ] As the previous reviewer suggested, Kindly match the design with the Figma design. You can try font weight and padding as well to match the design. Kindly do the change also for the get my  resume button. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![get](https://github.com/alexansaa/Personal_Portafolio/assets/70483835/8c80f5b8-aca2-46be-85a0-1f987aa4a69e) | ![getintouch_f](https://github.com/alexansaa/Personal_Portafolio/assets/70483835/72a53c85-e4c9-4f5b-9f54-5297c04f6f94)



Your design             |  Figma Design
:-------------------------:|:-------------------------:
![getmyresume](https://github.com/alexansaa/Personal_Portafolio/assets/70483835/7ff85bdb-9663-4581-8047-12809b1d793e) | ![Get my resume](https://github.com/alexansaa/Personal_Portafolio/assets/70483835/90370077-30f1-4019-8695-2043ce43540b)

Reply:
I have done these changes, but there is a bug with Linters. I can't give you a full Linter-checked code because of this bug. Many people are having the same issue. I'm going to send a new review request with this warning posted here so the next reviewer can notice this bug and let me proceed despite this issue.

Reply:
Hi @alexansaa, 👋🏿 
The Code review team has found the error to be somewhat universal. Your project is approved for now. Until a fix is found for the bug, Happy Coding. 🤝